README.md

```diff
@@ -6,8 +6,18 @@
 
 ## What is Cardano SL?
 
-**This codebase is now deprecated in favour of
-[cardano-node](https://github.com/input-output-hk/cardano-node),
-[cardano-db-sync](https://github.com/input-output-hk/cardano-db-sync),
-[cardano-graphql](https://github.com/input-output-hk/cardano-graphql) and
-[cardano-rest](https://github.com/input-output-hk/cardano-rest).**
+**This codebase is now deprecated in favour of more granular components.**
```

Comment:
It's not just more granular components, it's the new Shelley compatible implementations.
```suggestion
**This codebase is now deprecated in favour of the new Shelley-capable Cardano implementation.**

The new Cardano implementation consists of more granular components:
```